scripts/presets/replicaset-30.json

```diff
@@ -12,6 +12,7 @@
 				"noprealloc": true,
 				"nssize": 1,
 				"port": 3100,
+				"bind_ip_all": true,
```

Comment:
[`bind_ip_all` was introduced in 3.6](https://docs.mongodb.com/manual/reference/program/mongod/#cmdoption-mongod-bind-ip-all), so I don't think we should specify it for the configs using "30-release".

That said, I do think we should add it to all of the non-versioned configs, which will now be using "40-release". This includes all of the standalone JSON files apart from `standalone-30.json`.

Reply:
I suppose you can do `"bind_ip": "0.0.0.0,::"` for the 3.0 configs, per the documentation above.

Reply:
Done